View/AggregatedStageDamage/CalculatedStageDamageControl.xaml.cs

```diff
@@ -19,7 +19,57 @@ private void calculate_btn_Click(object sender, RoutedEventArgs e)
         {
             if (DataContext is CalculatedStageDamageVM vm)
             {
-                vm.CalculateCurves();
+                vm.ComputeCurves();
+            }
+        }
+
+        /// <summary>
+        /// This is to stretch the listview columns to fill the space as the user stretches the window.
```

Comment:
Is there a way to allow the columns to expand with the window by using wildcards in Xaml?